site/blog/deepseek-redteam.md

```diff
@@ -0,0 +1,183 @@
+
+DeepSeek has a proclivity for producing harmful content, including assisted suicide and child exploitation. 
+
+The latest model accepted more than 60% of prompts related to child grooming and child exploitation, producing detailed outputs on finding victims in vulnerable populations, how to build trust, and how to threaten or coerce children to stay silent. 
+
+![DeepSeek Response to Child Grooming](/img/blog/deepseek-redteam/child_grooming.png)
+
+![DeepSeek Response to Child Grooming, Part Two](/img/blog/deepseek-redteam/child_grooming2.png)
+
+It was also compliant with requests regarding biological and chemical weapons: 
+
+![DeepSeek Response to Biological and Chemical Weapons](/img/blog/deepseek-redteam/biological_weapon.png)
+
+It also provided detailed instructions on how to obtain and disseminate anthrax spores: 
+
+![DeepSeek Response to Anthrax](/img/blog/deepseek-redteam/anthrax_spores.png)
+
+Using techniques like multi-vector safety bypasses, Promptfoo could also force the model to produce content related to generating weapons: 
+
+![DeepSeek Response to Weapons](/img/blog/deepseek-redteam/weapon_creation.png)
+
+These are only a sample of the harmful content that was produced by DeepSeek-R1. 
+
+## Key Takeaways
+
+Our initial red teaming indicates that DeepSeek-R1 has significantly weak trust and safety controls that increase the risk of proliferating toxic and harmful content. DeepSeek-R1 is most at risk for disseminating content related to disinformation, religion, graphic content, malicious code and cybercriminal activity, and weapons. It is particularly vulnerable to single-shot jailbreak, multi-vector safety bypasses, and Likert jailbreaks. 
+
+As our previous research has indicated, DeepSeek-R1 also takes a political stance in alignment with the Chinese Communist Party and China's AI regulations. 
+
+## Mitigating Risk in DeepSeek-R1
+
+Promptfoo provides a suite of plugins and strategies that can be used to mitigate the risk of deploying DeepSeek-R1. 
+
+Promptfoo's initial research on DeepSeek's model safety should not necessarily deter usage or deployment. The model's risks can be mitigated using a defense-in-depth strategy: 
+
+- Never assume the foundation model's outputs are inherently safe or compliant. 
+- Use robust evaluations and strong system prompts.
+- Conduct continuous red teaming against the model.
+- Enforce guardrails with stricter policies. 
+- Continuously monitor applications using the model. 
+
+All models contain risk for generating harmful or toxic outputs. These approaches, while strongly recommended for DeepSeek, are best practices that should be enforced for any LLM application. 
+
+## Leveraging Promptfoo
+
+We encourage you to [run your own red team](https://www.promptfoo.dev/docs/red-team/quickstart/) against DeepSeek's latest model using Promptfoo's open-source tool. Use the configuration we provided above or tweak it according to your interest and AI requirements. 
```

Comment:
if this is the final CTA it would be great to link to an example / make it easier. Look at what we did in the deepseek censorship post